master/internal/api_checkpoint.go

```diff
@@ -129,12 +129,13 @@ func (a *apiServer) GetCheckpoint(
 	}
 
 	resp := &apiv1.GetCheckpointResponse{}
+	resp.Checkpoint = &checkpointv1.Checkpoint{}
 
-	ckpt, err := internaldb.GetCheckpoint(ctx, req.CheckpointUuid)
-	if err != nil {
-		return resp, errors.Wrapf(err, "error fetching checkpoint %s from database", req.CheckpointUuid)
+	if err := a.m.db.QueryProto(
```

Comment:
Can we add a comment here saying not to use Bun?

Reply:
Added. Thanks!